composition/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ if(NOT CMAKE_CXX_STANDARD)
 endif()
 
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-  add_compile_options(-Wall -Wextra)
+  add_compile_options(-Wall -Wextra -Wpedantic)
```

Comment:
This is actually throwing warnings on OSX because of https://github.com/ros/console_bridge/issues/55

Reply:
Worked around the clang console_bridge warnings in https://github.com/ros/class_loader/pull/80/commits/161568a8962c9616abbeaa1bbad751d16bbbbe85